midi2/src/utility.rs

```diff
@@ -97,15 +97,59 @@ mod timestamp {
 mod delta_clockstamp {
     use crate::detail::common_properties;
     use crate::utility;
+    use crate::ux::{u20, u4};
     pub const STATUS: u8 = 0b0100;
     #[midi2_proc::generate_message(Via(crate::utility::Utility), FixedSize, MinSizeUmp(1))]
     struct DeltaClockstamp {
         #[property(common_properties::UmpMessageTypeProperty<{utility::UMP_MESSAGE_TYPE}>)]
         ump_type: (),
         #[property(common_properties::ChannelVoiceStatusProperty<STATUS>)]
         status: (),
-        #[property(utility::DataProperty)]
-        time_data: u16,
+        #[property(DataProperty)]
+        time_data: u20,
+    }
+
+    struct DataProperty;
+
+    impl<B: crate::buffer::Ump> crate::detail::property::Property<B> for DataProperty {
+        type Type = u20;
+    }
+
+    impl<'a, B: crate::buffer::Ump> crate::detail::property::ReadProperty<'a, B> for DataProperty {
+        fn read(buffer: &'a B) -> Self::Type {
+            use crate::detail::BitOps;
+            use core::ops::BitOr;
```

Comment:
Why not use the `|` operator, and avoid the need to explicitly `use` this trait? The operator also reads better at the callsite imo.

Reply:
Fixed in commit c9c7480.  Agreed, nice and simple!